src/test/java/TestUtils.java

```diff
@@ -0,0 +1,90 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps;
+
+import static org.mockito.Mockito.when;
+
+import com.google.appengine.api.blobstore.BlobKey;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Key;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+import javax.servlet.http.HttpServletRequest;
+
+/** Class that contains helpful methods used for testing. */
+public final class TestUtils {
+  /** Private constructor to prevent instantiation. */
+  private TestUtils() {
+    throw new UnsupportedOperationException();
+  }
+
+  /* Add receipts to database for testing purposes. */
+  public static void addTestReceipts(DatastoreService datastore) {
+    addTestReceipt(datastore, 1, 123, 1045237591000L, new BlobKey("test"),
+        "img/walmart-receipt.jpg", 26.12, "walmart",
+        new HashSet<>(Arrays.asList("candy", "drink", "personal")), "");
+
+    addTestReceipt(datastore, 2, 123, 1513103400000L, new BlobKey("test"), "img/canes-receipt.jpg",
+        32.38, "raising cane's chicken fingers",
+        new HashSet<>(Arrays.asList("chicken", "drink", "lunch")), "");
+
+    addTestReceipt(datastore, 3, 123, 1560193140000L, new BlobKey("test"),
```

Comment:
For the parameters that aren't self-explanatory such as the ints, can you add a comment before each parameter. Example:
`addTestReceipt(datastore, /* id = */ 3, /* userId = */ 123, /* timestamp = */ 1560193140000L`)